### PR TITLE
update changelog for messaging and instanceID in M59

### PR DESCRIPTION
**Pull request by chliangGoogle**

### CODE REVIEW

File: Firebase/InstanceID/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-# Unreleased
-- [changed] Ensure tokenWithAuthorizedEntity:scope:options:handler method is refreshing token if token is not feshed any more. (#4096)
+# 2019-11-05 -- 4.2.7
+- [fixed] Fixed a crash in checkTokenRefreshPolicyWithIID: and ensure tokenWithAuthorizedEntity:scope:options:handler method is refreshing token if token is not feshed any more. (#4167)
```
Comment (morganchen12) [inline Firebase/InstanceID/CHANGELOG.md]: `feshed` -> `refreshed`?

Also, pleas encapsulate code (i.e. method names) in backticks ``

File: Firebase/InstanceID/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+- [changed] Update deprecated keychain access level. (#4172)
```
Comment (morganchen12) [inline Firebase/InstanceID/CHANGELOG.md]: nit: `Updated` instead of `Update`

File: Firebase/Messaging/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# 2019-11-05 -- v4.1.8
+- [changed] Move reliable message queue database operation off main thread. (#4053)
```
Comment (morganchen12) [inline Firebase/Messaging/CHANGELOG.md]: nit: `Moved` instead of `Move`

File: Firebase/InstanceID/CHANGELOG.md (hunk 2 above)
Comment (chliangGoogle) [reply Firebase/InstanceID/CHANGELOG.md]: Done.

File: Firebase/Messaging/CHANGELOG.md (hunk 3 above)
Comment (chliangGoogle) [reply Firebase/Messaging/CHANGELOG.md]: Done.

File: Firebase/InstanceID/CHANGELOG.md (hunk 1 above)
Comment (chliangGoogle) [reply Firebase/InstanceID/CHANGELOG.md]: Done.

**Pull request merged**